python/lsst/daf/persistence/butler.py

```diff
@@ -39,15 +39,9 @@
 import lsst.pex.logging as pexLog
 import lsst.pex.policy as pexPolicy
 from lsst.daf.persistence import StorageList, LogicalLocation, ReadProxy, ButlerSubset, ButlerDataRef, \
```

Comment:
This almost seems like you could `import *`, since it's the same module.  Otherwise it's a long list...